schrack/foobar.cpp

```diff
@@ -0,0 +1,536 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
```

Comment:
Is `foobar` class a leftover from playing with the tool or do you need those somewhere?